go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/ethereum/go-ethereum v1.10.4
+	github.com/ethereum/go-ethereum v1.10.23
```

Comment:
same as last PR, why is this necessary?
can you provide a link to the security issue related to v1.10.4?

Reply:
also, pls run `go mod` each time (and before your submit PR)

Reply:
Support for 1.18.5 is added in this version? Why we cannot upgrade Ethereum to this version?

Reply:
> also, pls run `go mod` each time (and before your submit PR)

Ok

Reply:
can you provide a link to the security issue related to v1.10.4， which was solved in v1.10.23?